internal/migrations/sql.go

```diff
@@ -195,7 +195,8 @@ var (
 	tableChangesetsMetadataSQL = `CREATE TABLE IF NOT EXISTS ` + migrationsSchemaName + `.changesets_metadata (
 		height INT PRIMARY KEY,
 		total_chunks INT, -- total number of chunks in the changeset
-		received INT -- number of chunks received
+		received INT, -- number of chunks received
+		prev_height INT -- height of the previous changeset
```

Comment:
We should probably have heights use bigint / int8.  A 4 byte integer is risky.

Reply:
oh yeah, makes sense.